monitor/monitor.go

```diff
@@ -233,6 +239,14 @@ type monitor struct {
 	invokeIO     *ioset.Forwarder
 	invokeCancel func()
 	attachedPid  atomic.Value
+
+	lastBuildResult *LastBuildResult
```

Comment:
variable name `lastBuildResult` is ok but the type returned from `RunMonitor` should be different. Maybe `MonitorBuildResult`?

Reply:
Fixed the type name to `MonitorBuildResult`.